notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>I would remove: This is because we cannot perform "general" calculations like logarithm, exponential, parallel transport, and calculating geodesics without first being equip with a metric</p><p> </p><p>Because, actually, we can! We just do not do it (yet!) in Geomstats.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>